SoftLayer/CLI/routes.py

```diff
@@ -104,6 +105,7 @@
     ('file:volume-count', 'SoftLayer.CLI.file.count:cli'),
     ('file:volume-detail', 'SoftLayer.CLI.file.detail:cli'),
     ('file:volume-duplicate', 'SoftLayer.CLI.file.duplicate:cli'),
+    ('file:volume-modify', 'SoftLayer.CLI.file.modify:cli'),
```

Comment:
This is minor, but it would be ideal if this line was moved just below the line for 'volume-list', so that the commands remain in alphabetical order.